yarn.lock

```diff
@@ -13903,7 +13903,7 @@ isobject@^4.0.0:
   resolved "https://registry.yarnpkg.com/isobject/-/isobject-4.0.0.tgz#3f1c9155e73b192022a80819bacd0343711697b0"
   integrity sha512-S/2fF5wH8SJA/kmwr6HYhK/RI/OkhD84k8ntalo0iJjZikgq1XFvR5M8NPT1x5F7fBwCG3qHfnzeP/Vh/ZxCUA==
 
-isomorphic-fetch@^2.1.1, isomorphic-fetch@^2.2.1:
+isomorphic-fetch@^2.1.1:
```

Comment:
Hm, what is this leftover?
is some other library using `isomorphic-fetch`?

Reply:
yup:

```
luis@pop-os:~/Documents/metabase$ yarn why isomorphic-fetch
yarn why v1.22.18
[1/4] Why do we have the module "isomorphic-fetch"...?
[2/4] Initialising dependency graph...
warning Resolution field "ansi-regex@5.0.1" is incompatible with requested version "ansi-regex@^2.0.0"
warning Resolution field "set-value@4.0.1" is incompatible with requested version "set-value@^2.0.0"
warning Resolution field "set-value@4.0.1" is incompatible with requested version "set-value@^2.0.1"
warning Resolution field "prismjs@1.27.0" is incompatible with requested version "prismjs@~1.25.0"
warning Resolution field "nth-check@2.0.1" is incompatible with requested version "nth-check@^1.0.2"
warning Resolution field "ansi-regex@5.0.1" is incompatible with requested version "ansi-regex@^2.0.0"
warning Resolution field "ansi-regex@5.0.1" is incompatible with requested version "ansi-regex@^4.0.0"
warning Resolution field "ansi-regex@5.0.1" is incompatible with requested version "ansi-regex@^6.0.1"
warning Resolution field "ansi-regex@5.0.1" is incompatible with requested version "ansi-regex@^2.0.0"
warning Resolution field "nanoid@3.1.31" is incompatible with requested version "nanoid@^3.3.3"
[3/4] Finding dependency...
[4/4] Calculating file sizes...
=> Found "isomorphic-fetch@2.2.1"
info Reasons this module exists
   - "@types#styled-system#styled-components#fbjs" depends on it
   - Hoisted from "@types#styled-system#styled-components#fbjs#isomorphic-fetch"
info Disk size without dependencies: "252KB"
info Disk size with unique dependencies: "884KB"
info Disk size with transitive dependencies: "6.18MB"
info Number of shared dependencies: 7
Done in 0.88s.
```

Reply:
To answer my own question:
```
=> Found "isomorphic-fetch@2.2.1"
info Reasons this module exists
   - "@types#styled-system#styled-components#fbjs" depends on it
   - Hoisted from "@types#styled-system#styled-components#fbjs#isomorphic-fetch"
info Disk size without dependencies: "252KB"
info Disk size with unique dependencies: "884KB"
info Disk size with transitive dependencies: "6.18MB"
info Number of shared dependencies: 7
```